src/history/base.js

```diff
@@ -252,11 +252,17 @@ export class History {
     // Default implementation is empty
   }
 
-  teardownListeners () {
+  teardown () {
```

Comment:
If we rename this method, it might make sense to rename the `setupListeners` method, too. https://github.com/vuejs/vue-router/blob/dfc289202703319cf7beb38d03c9258c806c4d62/src/history/base.js#L251